### PR TITLE
API keys: Add capability util

### PR DESCRIPTION
**Pull request by bduffany**
This will be used to convert between DB capabilities and proto capabilities.

---

**Version bump**: Minor

### CODE REVIEW

**Comment by bduffany** [comment]
Note to self: Change the base branch to `master` before merging

File: server/util/capabilities/capabilities.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+func FromInt(m int32) []akpb.ApiKey_Capability {
+	caps := []akpb.ApiKey_Capability{}
+	c := int32(1)
+	for m > 0 {
+		if m&1 > 0 {
+			caps = append(caps, akpb.ApiKey_Capability(c))
+		}
+		m >>= 1
+		c <<= 1
+	}
```
Comment (tylerwilliams) [inline server/util/capabilities/capabilities.go:16]: had to convince myself that this works but it does :)

another way is to say:

for _, cap := range allCaps {
  if cap & m > 0 {
    caps = append(caps, cap)
  }
return caps

**Review by tylerwilliams — APPROVED** [review]
LGTM

File: server/util/capabilities/capabilities.go (hunk 1 above)
Comment (bduffany) [reply server/util/capabilities/capabilities.go:16]: Ah yeah that looks safer; not as much casting. And shorter too! Done

**Pull request merged**